### PR TITLE
Fix missing CORS headers on RPC response.

### LINKED ISSUE #2028: Missing CORS response headers for RPC requests.
**Describe the bug**
The [json rpc handler](https://github.com/mopidy/mopidy/blob/master/mopidy/http/handlers.py#L192) from the `http` plugin sets the CORS headers when you do an `OPTIONS` request, but not for RPC request itself. This causes an issue with chrome, where says the CORS request is invalid.

**How to reproduce**
 Make sure that the domain you are using is whitelisted in the configuration:
```ini
[http]
allowed_origins = localhost:8080
```

Then, on  the domain that you whitelisted, run the following javascript code (running it in the console works fine):
```js
await window.fetch("http://localhost:6680/mopidy/rpc", {method: "POST", body: '{"jsonrpc":"2.0","id":null,"method":"core.playback.get_current_track"}', headers: new Headers({"Content-Type": "application/json"})})
```
This will result in a CORS error, because there is no `Access-Control-Allow-Origin` on the RPC response.

**Expected behaviour**
CORS requests should work for RPC on chrome.

**Environment**
Please complete the following information:

- Operating system: 
  - Arch Linux, Linux 5.14.18-lqx1-1-lqx

- [Running Mopidy](https://docs.mopidy.com/en/latest/running/) as a service or in the terminal?
  - I'm running it as a systemd service.

- Your config (output of `sudo mopidyctl config`)
  - <details>
     <summary>Config</summary>

     ```
    [core]
    cache_dir = /var/cache/mopidy
    config_dir = /etc/mopidy
    data_dir = /var/lib/mopidy
    max_tracklist_length = 10000
    restore_state = false
    
    [logging]
    verbosity = 0
    format = %(levelname)-8s %(asctime)s [%(process)d:%(threadName)s] %(name)s\n  %(message)s
    color = true
    config_file = /etc/mopidy/logging.conf
    
    [loglevels]
    mopidy_mpd.session = warning
    mopidy_spotify.playlists = critical
    mopidy_spotify.web = critical
    
    [audio]
    mixer = software
    mixer_volume =
    output = pulsesink server=127.0.0.1:4713
    buffer_time =
    
    [proxy]
    scheme =
    hostname =
    port =
    username =
    password =
    
    [file]
    enabled = true
    media_dirs =
      $XDG_MUSIC_DIR|Music
      ~/|Home
      /var/lib/mopidy/media|Media
    excluded_file_extensions =
      .directory
      .html
      .jpeg
      .jpg
      .log
      .nfo
      .pdf
      .png
      .txt
      .zip
    show_dotfiles = false
    follow_symlinks = true
    metadata_timeout = 1000
    
    [http]
    enabled = true
    hostname = 127.0.0.1
    port = 6680
    zeroconf = Mopidy HTTP server on $hostname
    allowed_origins =
      localhost:8080
    csrf_protection = true
    default_app = mopidy
    
    [m3u]
    enabled = true
    base_dir =
    default_encoding = latin-1
    default_extension = .m3u8
    playlists_dir = /var/lib/mopidy/playlists
    
    [softwaremixer]
    enabled = true
    
    [stream]
    enabled = true
    protocols =
      http
      https
      mms
      rtmp
      rtmps
      rtsp
    metadata_blacklist =
    timeout = 5000
    
    [ytmusic]
    enabled = true
    auth_json =
    auto_playlist_refresh = 60
    youtube_player_refresh = 15
    playlist_item_limit = 100
    subscribed_artist_limit = 100
    enable_history = true
    enable_liked_songs = true
    enable_mood_genre = true
    enable_scrobbling = true
    stream_preference =
      141
      251
      140
      250
      249
    verify_track_url = true
    
    [youtube]
    enabled = true
    allow_cache =
    youtube_api_key =
    search_results = 24
    playlist_max_videos = 24
    api_enabled = false
    channel_id =
    musicapi_enabled = false
    musicapi_cookie =
    autoplay_enabled = false
    strict_autoplay = false
    max_autoplay_length = 600
    max_degrees_of_separation = 3
    youtube_dl_package = youtube_dl
    
    [tunein]
    enabled = false  ; Extension disabled by user config.
    
    [spotify]
    enabled = true
    username = william.free@12oclockpoint.com
    password = ********
    client_id = f4c5f492-7d5d-4c86-93b9-2385fe2ef9bf
    client_secret = ********
    bitrate = 320
    volume_normalization = true
    private_session = false
    timeout = 5
    allow_cache = true
    allow_network = true
    allow_playlists = true
    search_album_count = 24
    search_artist_count = 12
    search_track_count = 50
    toplist_countries =
      US
      CA
    
    [spotify_web]
    enabled = false  ; Extension disabled by user config.
    
    [soundcloud]
    enabled = true
    explore_songs = 25
    auth_token = ********
    
    [somafm]
    enabled = true
    encoding = aac
    quality = highest
    dj_as_artist = true
    
    [scrobbler]
    enabled = true
    username = solonovamax
    password = ********
    
    [podcast]
    enabled = true
    browse_root = Podcasts.opml
    browse_order = desc
    lookup_order = asc
    cache_size = 64
    cache_ttl = 86400
    timeout = 10
    
    [pandora]
    enabled = false  ; Extension disabled by user config.
    
    [musicbox_webclient]
    enabled = true
    musicbox = false
    websocket_host =
    websocket_port = 6680
    on_track_click = PLAY_ALL
    
    [mpris]
    enabled = true
    bus_type = system
    
    [mpd]
    enabled = true
    hostname = 127.0.0.1
    port = 6600
    password =
    max_connections = 20
    connection_timeout = 60
    zeroconf = Mopidy MPD server on $hostname
    command_blacklist =
      listall
      listallinfo
    default_playlist_scheme = m3u
    
    [mowecl]
    enabled = true
    theme_type = light
    background_color = #fdf6e3
    text_color = #002b36
    primary_color = #268bd2
    seek_update_interval = 500
    search_history_length = 10
    disable_dnd = false
    small_screen = false
    key_play_pause = space+Control
    key_next_track = ArrowRight
    key_previous_track =
    key_rewind_track = ArrowLeft
    key_volume_up = ArrowUp
    key_volume_down = ArrowDown
    
    [mopify]
    enabled = true
    debug = false
    
    [mobile]
    enabled = true
    title = Mopidy Mobile on $hostname
    ws_url =
    
    [mixcloud]
    enabled = true
    users =
    tags =
    search_max = 20
    refresh_period = 600
    ignore_exclusive = true
    
    [local]
    enabled = true
    max_search_results = 100
    media_dir = /var/lib/mopidy/media
    scan_timeout = 1000
    scan_flush_threshold = 100
    scan_follow_symlinks = true
    included_file_extensions =
      .aac
      .flac
      .m4a
      .m4b
      .mp3
      .ogg
      .oga
      .mogg
      .opus
      .raw
      .sln
      .tta
      .wav
      .wv
      .webm
    excluded_file_extensions =
      .cue
      .directory
      .html
      .jpeg
      .jpg
      .log
      .nfo
      .pdf
      .png
      .txt
      .zip
    directories =
      Albums                  local:directory?type=album
      Artists                 local:directory?type=artist
      Composers               local:directory?type=artist&role=composer
      Genres                  local:directory?type=genre
      Performers              local:directory?type=artist&role=performer
      Release Years           local:directory?type=date&format=%25Y
      Tracks                  local:directory?type=track
      Last Week's Updates     local:directory?max-age=604800
      Last Month's Updates    local:directory?max-age=2592000
    timeout = 10
    use_artist_sortname = false
    album_art_files =
      *.jpg
      *.JPG
      *.jpeg
      *.JPEG
      *.png
      *.PNG
    
    [jamendo]
    enabled = false  ; Extension disabled by user config.
    
    [iris]
    enabled = true
    country = US
    locale = en_US
    snapcast_enabled = true
    snapcast_host = localhost
    snapcast_port = 1780
    spotify_authorization_url = https://jamesbarnsley.co.nz/iris/auth_spotify.php
    lastfm_authorization_url = https://jamesbarnsley.co.nz/iris/auth_lastfm.php
    genius_authorization_url = https://jamesbarnsley.co.nz/iris/auth_genius.php
    data_dir = /var/lib/mopidy/.iris
    
    [internetarchive]
    enabled = true
    base_url = http://archive.org
    collections =
      audio
      etree
      librivoxaudio
      audio_bookspoetry
      audio_tech
      audio_music
      audio_news
      audio_foreign
      audio_podcast
      audio_religion
    audio_formats =
      320Kbps MP3
      288Kbps MP3
      256Kbps MP3
      240Kbps MP3
      232Kbps MP3
      224Kbps MP3
      216Kbps MP3
      208Kbps MP3
      200Kbps MP3
      192Kbps MP3
      184Kbps MP3
      176Kbps MP3
      168Kbps MP3
      160Kbps MP3
      VBR MP3
      152Kbps MP3
      144Kbps MP3
      136Kbps MP3
      128Kbps MP3
      120Kbps MP3
      112Kbps MP3
      96Kbps MP3
      80Kbps MP3
      72Kbps MP3
      64Kbps MP3
    image_formats =
      PNG
      Animated GIF
      JPEG
      JPEG Thumb
      JPEG 2000
    browse_limit = 100
    browse_views =
      downloads desc|Views
      titleSorter asc|Title
      publicdate desc|Date Archived
      date desc|Date Published
      creatorSorter asc|Creator
    search_limit = 24
    search_order =
    cache_size = 128
    cache_ttl = 86400
    retries = 5
    timeout = 12
    
    [cd]
    enabled = true
    
    [bookmarks]
    enabled = true
    sync_period = 500
    max_bookmarks = 100
    max_bookmark_length = 100000
    max_store_items = 10
    max_store_item_length = 1000
    disable_limits = true
    
    [beets]
    enabled = true
    hostname = 127.0.0.1
    port = 8337
    
    [bandcamp]
    enabled = true
    discover_pages = 1
    collection_items = 50
    discover_genres =
      All
      Electronic
      Rock
      Metal
      Alternative
      Hip-Hop/Rap
      Experimental
      Punk
      Folk
      Pop
      Ambient
      Soundtrack
      World
      Jazz
      Acoustic
      Funk
      R&B/Soul
      Devotional
      Classical
      Reggae
      Podcasts
      Country
      Spoken Word
      Comedy
      Blues
      Kids
      Audiobooks
      Latin
    discover_tags =
      Outrun
      Future Funk
      Alternative Hip-Hop
      Tokyo, Japan
    image_sizes =
      10
      5
      2
    identity =
    
    [autoplay]
    enabled = true
    save_on_events =
    save_interval = 10
    tracklist.uris = auto
    tracklist.index = auto
    tracklist.consume = auto
    tracklist.random = auto
    tracklist.repeat = auto
    tracklist.single = auto
    playback.state = auto
    playback.time_position = auto
    mixer.volume = auto
    mixer.mute = auto
    
    [alsamixer]
    enabled = true
    card = 0
    control = Master
    min_volume = 0
    max_volume = 100
    volume_scale = cubic
    
    [advanced_scrobbler]
    enabled = false  ; Extension disabled by user config.
     ```
</details>

- Software versions (output of `sudo mopidyctl deps`)
  - <details>
     <summary>Dependencies</summary>
     
     ```
     Executable: /usr/bin/mopidy
     Platform: Linux-5.14.18-lqx1-1-lqx-x86_64-with-glibc2.33
     Python: CPython 3.9.7 from /usr/lib/python3.9
     Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       tornado: 6.1 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-MPRIS: 3.0.3 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       pydbus: 0.6.0 from /usr/lib/python3.9/site-packages
     Mopidy-Mobile: 1.10.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-Spotify-Web: 0.3.0 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       Mopidy-Spotify: 4.1.1 from /usr/lib/python3.9/site-packages
         Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
         Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
         requests: 2.26.0 from /usr/lib/python3.9/site-packages
           urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
           charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
           idna: 3.3 from /usr/lib/python3.9/site-packages
           certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
         setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
         pyspotify: 2.1.3 from /usr/lib/python3.9/site-packages
           cffi: 1.14.6 from /usr/lib/python3.9/site-packages
             pycparser: 2.20 from /usr/lib/python3.9/site-packages
           setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       spotipy: 2.19.0 from /usr/lib/python3.9/site-packages
         requests: 2.26.0 from /usr/lib/python3.9/site-packages
           urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
           charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
           idna: 3.3 from /usr/lib/python3.9/site-packages
           certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         six: 1.16.0 from /usr/lib/python3.9/site-packages
     Mopidy-Mopify: 1.7.3 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       ConfigObj: 5.1.0.dev0 from /usr/lib/python3.9/site-packages
         six: 1.16.0 from /usr/lib/python3.9/site-packages
     Mopidy-Advanced-Scrobbler: 2.1.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       pylast: 4.3.0 from /usr/lib/python3.9/site-packages
       music-metadata-filter: 2.0.1 from /usr/lib/python3.9/site-packages
       marshmallow: 3.14.0 from /usr/lib/python3.9/site-packages
       marshmallow-enum: 1.5.1 from /usr/lib/python3.9/site-packages
         marshmallow: 3.14.0 from /usr/lib/python3.9/site-packages
     Mopidy-TuneIn: 1.1.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-Cd: 0.5.1 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       musicbrainzngs: 0.7.1 from /usr/lib/python3.9/site-packages
     Mopidy-Autoplay: 0.2.3 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-Iris: 3.59.1 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-MPD: 3.2.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-Beets: 4.0.1 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-Pandora: 1.1.1 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       cachetools: 4.2.4 from /usr/lib/python3.9/site-packages
       pydora: 2.1.0 from /usr/lib/python3.9/site-packages
         blowfish: 0.6.1 from /usr/lib/python3.9/site-packages
         requests: 2.26.0 from /usr/lib/python3.9/site-packages
           urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
           charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
           idna: 3.3 from /usr/lib/python3.9/site-packages
           certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
     Mopidy-Local: 3.2.1 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       uritools: 3.0.2 from /usr/lib/python3.9/site-packages
     Mopidy-Podcast: 3.0.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       cachetools: 4.2.4 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       uritools: 3.0.2 from /usr/lib/python3.9/site-packages
     Mopidy-YouTube: 3.4 from /usr/lib/python3.9/site-packages
       beautifulsoup4: 4.9.3 from /usr/lib/python3.9/site-packages
         soupsieve: 2.2.1 from /usr/lib/python3.9/site-packages
       cachetools: 4.2.4 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       youtube-dl: 2021.6.6 from /usr/lib/python3.9/site-packages
       ytmusicapi: 0.19.4 from /usr/lib/python3.9/site-packages
         requests: 2.26.0 from /usr/lib/python3.9/site-packages
           urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
           charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
           idna: 3.3 from /usr/lib/python3.9/site-packages
           certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
     Mopidy-InternetArchive: 3.0.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       cachetools: 4.2.4 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       uritools: 3.0.2 from /usr/lib/python3.9/site-packages
     Mopidy-Bookmarks: 0.1.4 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       peewee: 3.14.8 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-Spotify: 4.1.1 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       pyspotify: 2.1.3 from /usr/lib/python3.9/site-packages
         cffi: 1.14.6 from /usr/lib/python3.9/site-packages
           pycparser: 2.20 from /usr/lib/python3.9/site-packages
         setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-Jamendo: 0.1.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       cachetools: 4.2.4 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     mopidy-ytmusic: 0.3.1 from /usr/lib/python3.9/site-packages
       ytmusicapi: 0.19.4 from /usr/lib/python3.9/site-packages
         requests: 2.26.0 from /usr/lib/python3.9/site-packages
           urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
           charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
           idna: 3.3 from /usr/lib/python3.9/site-packages
           certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       pytube: 10.9.3 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
     Mopidy-Scrobbler: 2.0.1 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       pylast: 4.3.0 from /usr/lib/python3.9/site-packages
     Mopidy-SomaFM: 2.0.2 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-MusicBox-Webclient: 3.1.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-SoundCloud: 3.0.1 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       requests: 2.26.0 from /usr/lib/python3.9/site-packages
         urllib3: 1.26.7 from /usr/lib/python3.9/site-packages
         charset-normalizer: 2.0.4 from /usr/lib/python3.9/site-packages
         idna: 3.3 from /usr/lib/python3.9/site-packages
         certifi: 2021.5.30 from /usr/lib/python3.9/site-packages
     Mopidy-ALSAMixer: 2.0.1 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       pyalsaaudio: 0.9.0 from /usr/lib/python3.9/site-packages
     Mopidy-Mowecl: 0.4.3 from /usr/lib/python3.9/site-packages
       Mopidy-Bookmarks: 0.1.4 from /usr/lib/python3.9/site-packages
         Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
         Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
         peewee: 3.14.8 from /usr/lib/python3.9/site-packages
         setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-Mixcloud: 3.1415.9265 from /usr/lib/python3.9/site-packages
       youtube-dl: 2021.6.6 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     Mopidy-Bandcamp: 1.1.5 from /usr/lib/python3.9/site-packages
       Mopidy: 3.2.0 from /usr/lib/python3.9/site-packages
       Pykka: 3.0.2 from /usr/lib/python3.9/site-packages
       setuptools: 57.4.0 from /usr/lib/python3.9/site-packages
     GStreamer: 1.18.5.0 from /usr/lib/python3.9/site-packages/gi
       Detailed information:
         Python wrapper: python-gi 3.42.0
         Relevant elements:
           Found:
             uridecodebin
             souphttpsrc
             appsrc
             alsasink
             osssink
             oss4sink
             pulsesink
             id3demux
             id3v2mux
             lamemp3enc
             mpegaudioparse
             mpg123audiodec
             vorbisdec
             vorbisenc
             vorbisparse
             oggdemux
             oggmux
             oggparse
             flacdec
             flacparse
             shout2send
           Not found:
             flump3dec
             mad
     ```
</details>

- Any specific hardware (e.g. an exotic USB soundcard)
  - No.

**Additional context**
I have a fix for this and I'll PR it in a bit. Just making this issue first.

This issue is similar to #1953.

### PR DESCRIPTION
**Pull request by solonovamax**
Fixes #2028.

Add CORS headers to RPC response.
This fixes a CORS error on chrome.

See #2028 for more information.

### CODE REVIEW

**Comment by djmattyg007** [comment]
What's preventing the CI pipeline from running for this PR?

**Comment by kingosticks** [comment]
No idea - too old? But I have changes for this anyway so it'll run when I push later.

**Comment by kingosticks** [comment]
Why is codecov so useless? I am mostly ignoring it at this point

File: tests/http/test_handlers.py [hunk 1]
```diff
@@ -93,6 +98,153 @@ def test_broadcast_to_client_without_ws_connection_present(self):
...
+        headers = {"Origin": "http://localhost", "Host": "localhost"}
+        conn = yield self.connection(headers=headers)
+        assert conn
+
+    @tornado.testing.gen_test
+    def test_bad_origin(self):
+        headers = {"Origin": "http://foobar", "Host": "localhost"}
+        with self.assertRaises(tornado.httpclient.HTTPClientError) as e:
+            _ = yield self.connection(headers=headers)
+        assert 403 == e.exception.code
```
Comment (djmattyg007) [inline tests/http/test_handlers.py:112]: This looks like another yoda expression that should be flipped.

Comment (kingosticks) [reply tests/http/test_handlers.py]: Cheers! Fixed

**Comment by djmattyg007** [comment]
I was kind of hoping we could get this issue resolved before doing further work on the CORS code:

https://github.com/mopidy/mopidy/issues/1966

The first step of my plan towards improving that situation was this PR, which has been waiting for review for a while now:

https://github.com/mopidy/mopidy/pull/2010

It would be great to get that PR moving, as there are many other follow-up changes, some of which I've already made and some of which others have made. For example, this PR also adds a `Float` config type, but I believe mine is superior. By merging #2010 I can then ask the developer to update their PR accordingly:

https://github.com/mopidy/mopidy/pull/2025

**Comment by kingosticks** [comment]
I'm just working through my long list of little things that personally bugged me. There are more CORS-related issues on the tracker, I'll visit yours next.

**Comment by kingosticks** [comment]
Oh I actually didn't mean to squash that one. Oops

**Comment by kingosticks** [comment]
And thanks @solonovamax for highlighting the issue and providing the original fix. I tweaked it a bit and added some comments explaining how the POST handler is designed to work w.r.t CORS because it wasn't totally obvious to me (after so long) and I wrote the thing!

**Comment by solonovamax** [comment]
Perfect! 
I wrote this fix because I had a little widget I made for OBS that connected to the mopidy server. But, since the OBS browser source used a chromium base which (for some reason) had no way to disable the CORS checking, I kept running into these errors
Great to see it merged!

**Pull request merged**